source/recovery.rst

```diff
@@ -99,43 +99,28 @@ which is based on three Percona XtraDB Cluster Pods.
 
    .. code-block:: bash
 
-      $ for i in $(seq 0 $(($(kubectl get pxc cluster1 -o jsonpath='{.spec.pxc.size}')-1))); do until [[ $(kubectl get pod cluster1-pxc-$i -o jsonpath='{.status.phase}') == 'Running' ]]; do sleep 10; done; kubectl exec cluster1-pxc-$i -- touch /var/lib/mysql/sst_in_progress; done
+      $ for i in $(seq 0 $(($(kubectl get pxc cluster1 -o jsonpath='{.spec.pxc.size}')-1))); do until [[ $(kubectl get pod cluster1-pxc-$i -o jsonpath='{.status.phase}') == 'Running' ]]; do sleep 10; done; kubectl exec cluster1-pxc-$i -c pxc -- touch /tmp/recovery-case; done
```

Comment:
do we still need debug image here? lines 50-90

Reply:
support wonts to have it, from our end we can avoid this doc at all.